appeals/web/src/server/views/app/components/file-uploader.component.njk

```diff
@@ -1,14 +1,7 @@
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
-{% macro extensionsHtml(params) %}
-<span> {{"Your file must be "}}
-{% for extension in params.allowedTypes %}
-	{% if loop.index !== params.allowedTypes.length %}{{ extension|upper }}{{", "}}
-	{% else %}{{"	or " }}{{ extension|upper }}.
-	{% endif %}
-{% endfor %}</span>
-{% endmacro %}
+{% macro extensionsHtml(params) %}<span>{{"Your file must be "}}{% for extension in params.allowedTypes %}{% if loop.index !== params.allowedTypes.length %}{{ extension|upper }}{{", "}}{% else %}{{"	or " }}{{ extension|upper }}.{% endif %}{% endfor %}</span>{% endmacro %}
```

Comment:
This is harder to read. I wonder if we can do the same thing with the whitespace control/tags?

e.g. `{%-` and `-%}`

https://mozilla.github.io/nunjucks/templating.html#whitespace-control